### PR TITLE
change in the amenties_rooms file and creation of model and controlle…

### PR DESCRIPTION
**Pull request by nessamoreno**
…r files.

### CODE REVIEW

File: app/Models/Amenitie.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Amenitie extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'air_conditionig',
```
Comment (mpauher) [inline app/Models/Amenitie.php]: change, the name it's bad -> air_conditioning

**Review by mpauher — COMMENTED** [review]
Models don't have the relations.
Delete controllers files, there are many controllers that probably we don't need.
Missing User model

File: app/Models/Customer.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+    protected $fillable =[
+        'identification',
+        'first_name',
+        'last_name',
+        'address',
+        'city',
+        'country',
+        'email',
+        'iphone',
```
Comment (mpauher) [inline app/Models/Customer.php]: change the name, -> "phone"

File: app/Models/Amenitie.php (hunk 1 above)
Comment (nessamoreno) [reply app/Models/Amenitie.php]: filled.

File: app/Models/Customer.php (hunk 2 above)
Comment (nessamoreno) [reply app/Models/Customer.php]: filled.

**Review by mpauher — APPROVED** [review]
Looks good Vane, thanks

**Review by mpauher — APPROVED** [review]

**Pull request merged**